### ISSUE TITLE
Add umd/es6 module exports

### ISSUE THREAD

**Issue by eordano** [zeit/swr#62]
This is a feature request.

Even though cjs+babel is the most common way to import modules, a number of projects out there use a es2015-style module imports. Like many libraries out there (like React), it'd be great if there's a `umd` export available for `swr`.

**Comment by sergiodxa** [comment]
Maybe using tsdx could help support this 🤔

**Comment by jackturnbull** [comment]
For what it's worth, I'm unable to consume this with rollup.js, although I can't wrap my head around why this is specifically. es2015 module would likely sidestep the issue from what I can see.

**Comment by mpint** [comment]
+1 from the Workers team at Cloudflare

**Comment by mpint** [comment]
@jackturnbull The problem here was that swr is using the spread operator in its published code, and although my webpack is configured to transpile the spread operator with babel, it ignores `node_modules` by default.

Whitelisting the library fixed it for me:

```
// webpack.config.js
webpackConfig.module.rules.push({
  test: /\/node_modules\/swr\/dist\/index.js/,
  use: [babelLoader],
})
```

**Comment by quietshu** [comment]
Hi all! We've landed ESM support in v0.1.12 (#140) and it's now tree-shakeable. Huge thanks to @yesmeck and everyone's help!

https://bundlephobia.com/result?p=swr@0.1.12
